openpype/hosts/nuke/plugins/publish/extract_review_baking_streams.py

```diff
@@ -25,6 +25,22 @@ class ExtractReviewDataMov(publish.Extractor):
     viewer_lut_raw = None
     outputs = {}
 
+    @classmethod
+    def apply_settings(cls, project_settings):
+        """just in case there are some old presets
+        in deprecated ExtractReviewDataMov Plugins
+        """
+        nuke_publish = project_settings["nuke"]["publish"]
+        deprecated_setting = nuke_publish["ExtractReviewDataMov"]
+        current_setting = nuke_publish["ExtractReviewDataBakingStreams"]
+        if not deprecated_setting["enabled"]:
+            if current_setting["enabled"]:
+                cls.viewer_lut_raw = current_setting["viewer_lut_raw"]
+                cls.outputs = current_setting["outputs"]
+        else:
+            cls.viewer_lut_raw = deprecated_setting["viewer_lut_raw"]
+            cls.outputs = deprecated_setting["outputs"]
```

Comment:
Would this be more readable?
```suggestion
        if deprecated_setting["enabled"]:
            # Use deprecated settings if they are still enabled
            cls.viewer_lut_raw = deprecated_setting["viewer_lut_raw"]
            cls.outputs = deprecated_setting["outputs"]
        elif current_setting["enabled"]:
            cls.viewer_lut_raw = current_setting["viewer_lut_raw"]
            cls.outputs = current_setting["outputs"]
```